samples/Sentry.Samples.Ios/Sentry.Samples.Ios.csproj

```diff
@@ -6,6 +6,7 @@
     <Nullable>enable</Nullable>
     <ImplicitUsings>true</ImplicitUsings>
     <SupportedOSPlatformVersion>10.0</SupportedOSPlatformVersion>
+    <SelfContained>true</SelfContained>
```

Comment:
What's the thinking behind this?

Reply:
Apparently, there was an implicit thing going on that `RuntimeIdetifier` would lead to a self-contained app (due to being runtime-specific). With .NET 8 that is no longer the case and requires the `SelfContained` property.